src/target/riscv/riscv-013.c

```diff
@@ -3396,311 +3396,532 @@ static int write_memory_abstract(struct target *target, target_addr_t address,
 	return result;
 }
 
+static uint32_t read_memory_progbuf_inner_startup_command(struct target *target)
+{
+	/* AC_ACCESS_REGISTER_POSTEXEC is used to trigger first stage of the
+	 * pipeline (memory -> s1) whenever this command is executed
+	 * (see the comment for read_memory_progbuf_inner_startup())
+	 */
```

Comment:
Just nitpicking - when there are longer comments, I prefer to use proper sentences for readability (capital letter at the beginning, period at the end).

```suggestion
	/* AC_ACCESS_REGISTER_POSTEXEC is used to trigger first stage of the
	 * pipeline (memory -> s1) whenever this command is executed.
	 * (See the comment for read_memory_progbuf_inner_startup().)
	 */
```

Reply:
Addressed.